packages/components/src/sites-table/site-status.tsx

```diff
@@ -0,0 +1,37 @@
+import { useI18n } from '@wordpress/react-i18n';
+
+export const siteStatuses = [ 'coming-soon', 'private', 'public' ] as const;
+
+export type SiteStatus = typeof siteStatuses[ number ];
```

Comment:
I wonder if `SiteStatus` is the correct name?

There are lots of attributes on the site object that are roughly related to its "status". This code reflects some of that data, but not all of it.

While it would be a bit of refactoring, `SiteLaunchStatus`, `getSiteLaunchStatus`, etc. would be more specific naming.

Reply:
That makes sense. Will do that tomorrow!